### DISCUSSION TITLE
destination leaving port 25

### DISCUSSION THREAD

**Discussion by maiquelsolutio** [postalserver/postal#1981] [Help with using Postal]
In 1 year I didn't have any problems with project delivery, recently we started having delivery problems with a specific regional provider, after a lot of information exchange, we noticed that the output present in smtp_sender.log, informs sending via IP port PROVIDER: 25 .

This doesn't happen on other providers, and I don't technically understand why.

How can I force the send out by 587?

Why only this provider has this behavior when others go out normally through another port?

Grateful for the attention.

**Comment by mitmelon** [comment]
Your provider must have blocked port 25. This days. Cloud provider block this port to prevent spamings. You will have to contact your provider to unblock port 25 for you.

**Reply by willpower232** [reply]
Yeah either your server company is blocking it or the "specific regional provider" is blocking you for some reason. Changing to port 587 won't do anything other than make your own setup more complicated.
